
PLAY [reco_release] ************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************************************************************************
ok: [10.11.5.113]
ok: [10.11.5.111]
ok: [10.11.5.115]
ok: [10.11.5.112]
ok: [10.11.5.116]
ok: [10.11.5.117]
ok: [10.11.5.118]
ok: [10.11.5.119]
ok: [10.11.5.114]
ok: [10.11.5.120]
ok: [10.11.5.121]
ok: [10.11.5.122]
ok: [10.11.5.123]
ok: [10.11.5.124]
ok: [10.11.5.125]
ok: [10.11.5.126]
ok: [10.11.5.127]
ok: [10.11.5.128]
ok: [10.11.5.129]
ok: [10.11.5.130]
ok: [10.11.5.131]
ok: [10.11.5.132]
ok: [10.11.5.133]
ok: [10.11.5.134]

TASK [reco_release : pull reco images 123] *************************************************************************************************************************************************************************************************
fatal: [10.11.5.112]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.379133", "end": "2018-11-14 09:48:11.180209", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:04.801076", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.115]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.493734", "end": "2018-11-14 09:48:11.293592", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:04.799858", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.111]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.607077", "end": "2018-11-14 09:48:11.423020", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:04.815943", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.113]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:08.689560", "end": "2018-11-14 09:48:13.496275", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:04.806715", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.116]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.766940", "end": "2018-11-14 09:48:15.329262", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:11.562322", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.118]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.646497", "end": "2018-11-14 09:48:15.419083", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:11.772586", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.117]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.819980", "end": "2018-11-14 09:48:15.483190", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:11.663210", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.119]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.507549", "end": "2018-11-14 09:48:17.358884", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:13.851335", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.114]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:12.979783", "end": "2018-11-14 09:48:17.816561", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:04.836778", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: TLS handshake timeout", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: TLS handshake timeout"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.121]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.506428", "end": "2018-11-14 09:48:19.265505", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:15.759077", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.122]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.539531", "end": "2018-11-14 09:48:19.346528", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:15.806997", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.120]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.203648", "end": "2018-11-14 09:48:19.867782", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:15.664134", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.123]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.716679", "end": "2018-11-14 09:48:21.457772", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:17.741093", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.124]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.221242", "end": "2018-11-14 09:48:22.353583", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:18.132341", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.125]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:04.539861", "end": "2018-11-14 09:48:24.174964", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:19.635103", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.127]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.210648", "end": "2018-11-14 09:48:26.442085", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:20.231437", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.126]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:07.760688", "end": "2018-11-14 09:48:27.512550", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:19.751862", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.128]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:08.275252", "end": "2018-11-14 09:48:30.119477", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:21.844225", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.129]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:07.780328", "end": "2018-11-14 09:48:30.525909", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:22.745581", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.130]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.776227", "end": "2018-11-14 09:48:31.280302", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:24.504075", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.132]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:05.001498", "end": "2018-11-14 09:48:32.754598", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:27.753100", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.131]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:06.112332", "end": "2018-11-14 09:48:32.932069", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:26.819737", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.133]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.729744", "end": "2018-11-14 09:48:34.133548", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:30.403804", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
fatal: [10.11.5.134]: FAILED! => {"changed": true, "cmd": "docker pull 123", "delta": "0:00:03.499438", "end": "2018-11-14 09:48:34.369559", "msg": "non-zero return code", "rc": 1, "start": "2018-11-14 09:48:30.870121", "stderr": "Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'", "stderr_lines": ["Error response from daemon: pull access denied for 123, repository does not exist or may require 'docker login'"], "stdout": "Using default tag: latest", "stdout_lines": ["Using default tag: latest"]}
	to retry, use: --limit @/home/op/cheng/autohdmap_deploy/reco_release.retry

PLAY RECAP *********************************************************************************************************************************************************************************************************************************
10.11.5.111                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.112                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.113                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.114                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.115                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.116                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.117                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.118                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.119                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.120                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.121                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.122                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.123                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.124                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.125                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.126                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.127                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.128                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.129                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.130                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.131                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.132                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.133                : ok=1    changed=0    unreachable=0    failed=1   
10.11.5.134                : ok=1    changed=0    unreachable=0    failed=1   

[Finish] Failed to deploy [123]